### PR TITLE
Add titles to menu entries

### PR DESCRIPTION
**Pull request by willdurand**
screen shot 2018-01-30 at 13 46 23

### CODE REVIEW

**Comment by willdurand** [comment]
@jmaupetit so it seems the fix for the fixtures did not make it..

File: templates/base.html [hunk 1]
```diff
@@ -29,29 +29,33 @@
           <li>
             <a class="admin" href="{% url 'admin:index' %}">
               <i class="fa fa-wrench" aria-hidden="true"></i>
+              {% trans "Django admin" %}
```
Comment (jmaupetit) [inline templates/base.html]: `s/Django admin/Admin`

**Comment by jmaupetit** [comment]
I think the number of experiments / pixel sets matters.

**Comment by jmaupetit** [comment]
BTW this is a great improvement.

File: templates/base.html (hunk 1 above)
Comment (willdurand) [reply templates/base.html]: I only capitalized the first word of each title.

Comment (jmaupetit) [reply templates/base.html]: No, I mean remove `Django`, just let `Admin`

Comment (willdurand) [reply templates/base.html]: haha I was thinking about that when your comment appeared 🌟

**Pull request merged**